Update lob-apps-macos-agent.md (#4068)
MicrosoftDocs/memdocs: memdocs/intune/apps/lob-apps-macos-agent.md

## How to turn off usage data sent to Microsoft for shell scripts?

>  To turn off usage data sent to Microsoft from the Intune management agent, open Company Portal and select **Menu** then **Preferences** and uncheck *"allow Microsoft to collect usage data"*. This will turn off usage data sent for both the agent and Company Portal.

Reviewer: ```suggestion
 To turn off usage data sent to Microsoft from the Intune management agent:

-  Open Company Portal.

- Select **Menu**.

- Select **Preferences**.

- Clear the **Allow Microsoft to collect usage data** checkbox. 

This turns off usage data sent for both the agent and Company Portal.

```

>  To turn off usage data sent to Microsoft from the Intune management agent, open Company Portal and select **Menu** then **Preferences** and uncheck *"allow Microsoft to collect usage data"*. This will turn off usage data sent for both the agent and Company Portal.

Reviewer: ```suggestion
 To turn off usage data sent to Microsoft from the Intune management agent, open Company Portal, point to **Menu**, select **Preferences**, and then clear the **allow Microsoft to collect usage data** checkbox. This turns off usage data sent for both the agent and Company Portal.
```
